src/components/CandidateForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import style from "./CandidateForm.module.css";
+import Button from "./Button";
+
+function CandidateForm(props) {
+  const [items, setItem] = useState(props.inputs);
```

Comment:
A cleaner way would be to destructure 'inputs and action' from props. e.g. `const {inputs, actions}  = props`